### ISSUE TITLE
Feature Request - ability to launch update from the web gui

### ISSUE THREAD

**Issue by rtwright68** [wh1te909/tacticalrmm#307]
I do love and use SSH all the time, lol.  But, it would be cool to allow an update to be launched directly from the web GUI.

**Comment by dinger1986** [comment]
I think this is something everyone would like to see and should be implemented for now whilst there's breaking changes in some updates doing it at the terminal is best. Probably should be in when tactical is stable.

**Comment by rtwright68** [comment]
Sounds good!

**Comment by wh1te909** [comment]
Yes just what @dinger1986 said, forcing ssh for now because sometimes need user intervention from bash script for breaking changes.

**Comment by silversword411** [comment]
Is this about updating TRMM agent, or updating OS?

**Comment by Sander-VdB** [comment]
I read this as updating the TRMM server through the web GUI.
If so: it's something I want too.

If possible update the whole underlying OS so I don't have to SSH into it at all. 
(And maybe add an option to auto update at a specific time but that might be it's own feature request)

**Comment by dinger1986** [comment]
> I read this as updating the TRMM server through the web GUI.
> 
> If so: it's something I want too.
> 
> 
> 
> If possible update the whole underlying OS so I don't have to SSH into it at all. 
> 
> (And maybe add an option to auto update at a specific time but that might be it's own feature request)

Yeah it's for the server.

It'll be implemented when breaking changes aren't happening.

You can update the rmm by putting the agent on it although that could lead to other potential issues

Auto update could be another feature request but maybe the devs will pick it up off of here as and when update form the web is implemented.
